pyqtgraph/graphicsItems/GridItem.py

```diff
@@ -1,33 +1,99 @@
 from ..Qt import QtGui, QtCore
-from .UIGraphicsItem import *
+from .UIGraphicsItem import UIGraphicsItem
 import numpy as np
 from ..Point import Point
 from .. import functions as fn
+from .. import getConfigOption
 
 __all__ = ['GridItem']
 class GridItem(UIGraphicsItem):
     """
     **Bases:** :class:`UIGraphicsItem <pyqtgraph.UIGraphicsItem>`
-    
+
     Displays a rectangular grid of lines indicating major divisions within a coordinate system.
     Automatically determines what divisions to use.
     """
-    
-    def __init__(self):
+
+    def __init__(self, pen='default', textColor='default'):
         UIGraphicsItem.__init__(self)
         #QtGui.QGraphicsItem.__init__(self, *args)
         #self.setFlag(QtGui.QGraphicsItem.ItemClipsToShape)
         #self.setCacheMode(QtGui.QGraphicsItem.DeviceCoordinateCache)
-        
+
+        self.opts = {}
+
+        if pen == 'default':
+            self.setPen()
+        else:
+            self.setPen(pen)
+
+        if textColor == 'default':
+            self.setTextColor()
+        else:
+            self.setTextColor(textColor)
+
+        self.setTickSpacing(x=[None, None, None], y=[None, None, None])
```

Comment:
This should probably also be specifiable in that method.

Reply:
It is probably not that important to be able to specify the tick spacing when calling the constructor.

Reply:
This is probably not that important, resolving.